workshop/content/japanese/30-python/50-table-viewer/200-install.md

```diff
@@ -0,0 +1,23 @@
++++
+title = "ライブラリのインストール"
+weight = 200
++++
+
+## pip install
```

Comment:
Please update this page to be consistent with the english version. A snippet has been added and the content has been slightly modified to make things more clear